ts-client/src/amm/utils.ts

```diff
@@ -994,15 +994,15 @@ export async function createMint(
   return { tx: transaction, mintAccount };
 }
 
-export const calculateLockAmounts = (amount: BN, feeWrapperPercent?: Decimal) => {
-  const safeFeeWrapperPercent = feeWrapperPercent?.gt(new Decimal(0))
-    ? Decimal.min(feeWrapperPercent, new Decimal(1))
-    : new Decimal(0);
+export const calculateLockAmounts = (amount: BN, feeWrapperRatio = new Decimal(0)) => {
+  if (feeWrapperRatio?.lt(0) || feeWrapperRatio?.gt(1)) {
```

Comment:
do we need `feeWrapperRatio ?` because we already set default value as Decimal(0)